metabill/src/metabill/core.clj

```diff
@@ -25,7 +25,12 @@
     d))
 
 (defn load-build-meta-data []
-  (edn/read-string (slurp (io/resource metabill-filename))))
+  (let [resource-file (io/resource metabill-filename)]
+    (if resource-file
+      (try
+        (edn/read-string (slurp resource-file))
+        (catch Exception _ nil))
```

Comment:
I don’t think it is good to ignore all `Exceptions and squash the result to `nil`. :thinking: 

Please fix it again to handle exceptions separately, such as EDN parse error or some other error. In addition, please output warning logs to the stderr.